packages/react/src/components/NumberInput/NumberInput.js

```diff
@@ -139,6 +139,10 @@ class NumberInput extends Component {
      * `true` to use the mobile variant.
      */
     isMobile: PropTypes.bool,
+    /**
+     * Specify the size of the Text Input. Currently supports either `sm` or `xl` as an option.
+     */
+    size: PropTypes.oneOf(['sm', 'xl']),
```

Comment:
```suggestion
    /**
     * Specify the size of the Number Input. Currently supports either `sm` or `xl` as an option.
     */
    size: PropTypes.oneOf(['sm', 'xl', '']),
```

Reply:
Per this conversation https://github.com/carbon-design-system/carbon/pull/5472#issuecomment-592221372, it may be best to have a default of `lg` instead? Given that's the intended true default for all of these sizes? 🤔